internal/suites/suite_pathprefix_test.go

```diff
@@ -14,6 +14,10 @@ func NewPathPrefixSuite() *PathPrefixSuite {
 	return &PathPrefixSuite{RodSuite: new(RodSuite)}
 }
 
+func (s *PathPrefixSuite) TestCheckEnv() {
```

Comment:
I suspect this file should actually change after we've merged in the suite environment changes first?

Reply:
Yep, makes sense. Did they look good to you?